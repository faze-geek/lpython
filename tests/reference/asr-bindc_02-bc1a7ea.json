{
    "basename": "asr-bindc_02-bc1a7ea",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/bindc_02.py",
    "infile_hash": "b63020f2df6e1cdbc6bd3fd8ef551b6b62fda00345bbeb12c7b94305",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-bindc_02-bc1a7ea.stderr",
    "stderr_hash": "671b57ec96c81682f9f340a3c0ea1ad45ed985ea61450bc48d97af6b",
    "returncode": 2
}