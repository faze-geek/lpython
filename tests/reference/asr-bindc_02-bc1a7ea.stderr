semantic error: 'i16[3]' object does not support item assignment
  --> tests/../integration_tests/bindc_02.py:12:5
   |
12 |     y[0] = i16(1)
   |     ^^^^^^^^^^^^^ 
